### DISCUSSION TITLE
interfaceをインタフェースに訳語統一

### DISCUSSION THREAD

**Discussion by KenichiroTanaka** [pgsql-jp/jpug-doc#3042] [NGとする語]
## 問題点

interfaceの訳語が統一されていません。
## 背景

$ find . -name ".sgml"|xargs grep "インターフェース"|wc -l
3
$ find . -name ".sgml"|xargs grep "インターフェイス"|wc -l
21
$ find . -name ".sgml"|xargs grep "インターフェス"|wc -l
0
$ find . -name ".sgml"|xargs grep "インタフェース"|wc -l
159

## 解決方法

（解決方法を書いてください）

## 注意点

（注意点、懸念事項を書いてください）

**Comment by KenichiroTanaka** [comment]
数も多いですし、
http://www.jp.freebsd.org/man-jp/docs/guideline.html でも「インタフェース」がお薦めにはなっています。

インタフェースで統一しようと思いますが反対意見があればお知らせください。

**Comment by noborus** [comment]
一旦インタフェースで良いと思います。

**Comment by noborus** [comment]
統一されたのでクローズします。
